boards/arm/verdin_imx8mp_m7/verdin_imx8mp_m7_itcm_defconfig

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (c) 2023 Toradex
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_SOC_SERIES_IMX8ML_M7=y
+CONFIG_SOC_MIMX8ML8=y
+CONFIG_BOARD_VERDIN_IMX8MP_M7=y
+CONFIG_CLOCK_CONTROL=y
+CONFIG_UART_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
```

Comment:
Is `CONFIG_UART_INTERRUPT_DRIVEN` needed as part of the minimal set of Kconfigs to enable this platform? If not, I would like to drop it here

Reply:
It's not necessary. I am dropping it.